web-console/knikubevirt/migrate-vm/migrate-vm.md

```diff
@@ -0,0 +1,19 @@
+# Migrate VM
+
+Clicking the action button next to any Virtual Machine in the List will present the user with the option to migrate the VM to another node.
+
+![Selecting a VM](img/1-0-list.png)
+
+A modal will be presented where the user can choose which Node the VM should be migrated to. 
+
+![Modal migrate](img/1-2-modal-migrate.png)
+
+![Modal view](img/1-1-modal-info.png)
+
+After the migration is kicked off, the user will see a toast notification letting them know it has started. The state for the VM will also be reflected in the list view.
+
+![Toast migration started](img/1-3-list-notification-started.png)
```

Comment:
How long to migrations usually take? I can see the toast definitely being useful on the completion side if this is a long process where the user would want to go do another task and then be notified once the migration is complete. If the user took the action to click migrate and then they see the status change to migrate...do they need the toast to explain the migration started as well? The reason I pick on this is because we previously had a LOT of toasts for when things started and finished and users got annoyed by the extra notifications.

Reply:
@beanh66 that's a good point. I think have just a confirmation of success/failure is needed. They would be aware that the migration has started based on their own actions.